src/ZenstruckMessengerTestBundle.php

```diff
@@ -65,7 +71,19 @@ public function process(ContainerBuilder $container): void
                 }
             }
 
-            $registry->addMethodCall('register', [$name, new Reference($id)]);
+            $transportRegistry->addMethodCall('register', [$name, new Reference($id)]);
+        }
+
+        $busRegistry = $container->getDefinition('zenstruck_messenger_test.bus_registry');
+        foreach ($container->findTaggedServiceIds('messenger.bus') as $id => $tags) {
+            $name = "$id.test-bus";
+            $busRegistry->addMethodCall('register', [$name, new Reference($name)]);
```

Comment:
```suggestion
            $busRegistry->addMethodCall('register', [$id, new Reference($name)]);
```

I believe this will remove the need to suffix the name in TestBusRegistry.